app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::UsersController < Admin::BaseController
+  def index
+
```

Comment:
Clicking on all users as an admin lead to an error, now it is just a black page.